components/automate-chef-io/content/docs/backup.md

```diff
@@ -313,3 +288,42 @@ Are you sure you want to continue? (y/n)
 y
 Success: Backups deleted
 ```
+
+## Troubleshooting
+
+### Update the FQDN
```

Comment:
[nit] I would not put this under troubleshooting; it's pretty normal to take a backup of an application on a host with one FQDN and restore it on host with a different FQDN, changing the things that need changing. I see it as a branch in the process much like single-file archives.

Reply:
Thanks! Let's merge it for now.